feat: Redirect Merchant Messages [skip ci]

cmkohnen opened the pull request:
Filter Merchant Messages the likes of "Scroll Merchant: Thank you for your business. Come again!", as they can spam the chat quite quickly when buying multiple items

kristofbolyai reviewed (commented):
Can you move this into Message Redirect Feature? (Actually, I don't remember the feature name, but you will see what I am talking about. I want to make this change redirect to notification overlay, by default.

cmkohnen committed:
feat: Redirect Merchant Messages

Redirect Merchant Messages the likes of "Scroll Merchant: Thank you for
your business. Come again!", as they can spam the chat quite quickly
when buying multiple items

Signed-off-by: Christoph Matthias Kohnen <christoph.kohnen@disroot.org>

cmkohnen committed:
ci: spotless formatting

kristofbolyai reviewed (approved):
Thanks!
